two-factor.php

```diff
@@ -4,24 +4,29 @@
  * Plugin URI: http://github.com/georgestephanis/two-factor/
  * Description: A prototype extensible core to enable Two-Factor Authentication.
  * Author: George Stephanis
- * Version: 0.1-dev
+ * Version: 0.1-dev-20161216
  * Author URI: http://stephanis.info
  * Network: True
  * Text Domain: two-factor
  */
 
-/**
- * Shortcut constant to the path of this file.
- */
-define( 'TWO_FACTOR_DIR', plugin_dir_path( __FILE__ ) );
+// Disable if in dev mode.
+if ( JETPACK_DEV !== true ) {
```

Comment:
This would need a `defined()` check first. Also, I think the constant is called `JETPACK_DEV_DEBUG`.